specs/nytimes/nytimes-best-sellers.html

```diff
@@ -3,30 +3,16 @@
     <title>NYTimes Best Sellers</title>
   </head>
   <body>
-    <style>
```

Comment:
This whole `<style>` block is not necessary anymore, since we don't plan to tweak the look-and-feel of the distilled HTML. It will boil down to JSON only.